docker-compose.yml

```diff
@@ -1,53 +1,83 @@
 version: '2'
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=secret
+    volumes:
+      - postgresql_data:/var/lib/postgresql/data
 
   elasticsearch:
-   image: elasticsearch
+    image: elasticsearch:5.2
+    volumes:
+      - elasticsearch_data:/usr/share/elasticsearch/data
 
   rabbitmq:
-     image: rabbitmq
+     image: rabbitmq:3.6
+     volumes:
+      - rabbitmq_data:/var/lib/rabbitmq
 
   django:
     image: geonode/django
     links:
-      - postgres
+      - database
       - elasticsearch
       - rabbitmq
     command: django-admin.py runserver 0.0.0.0:8000 --settings=geonode.settings
-    env_file:
```

Comment:
I'd rather leave env file and keep configuration for production and development separated